www/chromium/files/extra-patch-libc++-new

```diff
@@ -20,68 +12,45 @@
  
  def check_call(cmd, **kwargs):
    logging.debug('Running: %s', ' '.join(cmd))
-@@ -449,6 +451,7 @@
-       'base/metrics/histogram_base.cc',
-       'base/metrics/histogram_functions.cc',
-       'base/metrics/histogram_samples.cc',
-+      'base/metrics/histogram_snapshot_manager.cc',
-       'base/metrics/metrics_hashes.cc',
-       'base/metrics/persistent_histogram_allocator.cc',
-       'base/metrics/persistent_memory_allocator.cc',
-@@ -534,7 +537,7 @@
-       'base/trace_event/heap_profiler_allocation_context_tracker.cc',
-       'base/trace_event/heap_profiler_allocation_register.cc',
-       'base/trace_event/heap_profiler_event_filter.cc',
--      'base/trace_event/heap_profiler_event_writer.cc',
-+      'base/trace_event/heap_profiler_heap_dump_writer.cc',
-       'base/trace_event/heap_profiler_serialization_state.cc',
-       'base/trace_event/heap_profiler_stack_frame_deduplicator.cc',
-       'base/trace_event/heap_profiler_type_name_deduplicator.cc',
-@@ -627,7 +630,7 @@
+@@ -576,7 +577,6 @@
+       'base/trace_event/trace_log_constants.cc',
+       'base/trace_event/tracing_agent.cc',
+       'base/unguessable_token.cc',
+-      'base/value_iterators.cc',
+       'base/values.cc',
+       'base/vlog.cc',
+   ])
+@@ -633,6 +633,44 @@
          'cflags': cflags + ['-DHAVE_CONFIG_H'],
      }
  
--  if is_linux or is_aix:
-+  if is_linux or is_aix or is_bsd:
-     ldflags.extend(['-pthread'])
- 
-     static_libraries['xdg_user_dirs'] = {
-@@ -642,16 +645,44 @@
-         'base/memory/shared_memory_tracker.cc',
-         'base/nix/xdg_util.cc',
-         'base/process/internal_linux.cc',
--        'base/process/memory_linux.cc',
-+        #'base/process/memory_linux.cc',
-         'base/process/process_handle_linux.cc',
-         'base/process/process_info_linux.cc',
--        'base/process/process_iterator_linux.cc',
--        'base/process/process_linux.cc',
--        'base/process/process_metrics_linux.cc',
-+        #'base/process/process_iterator_linux.cc',
-+        #'base/process/process_linux.cc',
-+        #'base/process/process_metrics_linux.cc',
-         'base/strings/sys_string_conversions_posix.cc',
--        'base/sys_info_linux.cc',
-+        #'base/sys_info_linux.cc',
-         'base/threading/platform_thread_linux.cc',
-     ])
-+    if is_bsd:
-+        libs.extend(['-lexecinfo', '-lkvm'])
-+        ldflags.extend(['-pthread'])
-+    
++  if is_bsd:
++    libs.extend(['-lexecinfo', '-lkvm'])
++    ldflags.extend(['-pthread'])
++    include_dirs += ["/usr/local/include"]
```

Comment:
Edge case, this should be expanded from ${PREFIX} in case someone installs packages in e.g. /opt